src/containerapp/HISTORY.rst

```diff
@@ -5,6 +5,11 @@ Release History
 upcoming
 ++++++
 * 'az containerapp debug': Open an SSH-like interactive shell within a container app debug console.
+* 'az containerapp create': New target-label option for Labels revision mode.
+* 'az containerapp update': New revisions-mode and target-label options for Labels revision mode.
+* 'az containerapp up': New revisions-mode and target-label options for Labels revision mode.
```

Comment:
Should the customer use sub command group for this property?

Reply:
?

Reply:
These are covered by the new tests.